com.unity.render-pipelines.universal/Runtime/Passes/FinalBlitPass.cs

```diff
@@ -108,7 +108,7 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                         cameraTarget,
                         RenderBufferLoadAction.Load,
                         RenderBufferStoreAction.Store,
-                        ClearFlag.None,
+                        ClearFlag.Depth,
```

Comment:
I don't follow why we are changing this. Could this maybe introduce regression for users that might read depend on depth after final blit pass?